app/assets/v2/js/pages/dashboard.js

```diff
@@ -1,6 +1,22 @@
 /* eslint-disable no-loop-func */
 // helper functions
-var sidebar_keys = [ 'experience_level', 'project_length', 'bounty_type', 'bounty_filter', 'network', 'idx_status' ];
+var technologies = [
```

Comment:
@PixelantDesign @mbeacom  we are hardcoding this for now !! Cause I believe no alternative was given during the offline discussion